system/reports/assets/simple.cfm

```diff
@@ -171,8 +168,11 @@
 <script>
 $( document ).ready( function(){
 	// spec toggler
-	$( "span.specStatus" ).click( function(){
-		toggleSpecs( $( this ).attr( "data-status" ), $( this ).attr( "data-bundleid" ) );
+	$("span.specStatus").click(function() {
+		$(this).parent().children().removeClass("active");
```

Comment:
Can you please use the right spacing for whitespace.

```
// Not this
$(this)
removeClass("active")

// But this
$( this )
removeClass( "active" )

```

Reply:
Fixed.